packages/client/hmi-client/src/model-representation/petrinet/petrinet-service.ts

```diff
@@ -94,16 +95,19 @@ export const convertToIGraph = (amr: Model) => {
 		const typeMap = amr.semantics?.typing?.type_map.find(
 			(map) => map.length === 2 && transition.id === map[0]
 		);
+
+		const targetNode = amr.semantics?.ode.rates.find((node) => transition.id === node.target);
+
```

Comment:
You are not trying to find a "node", would be clearer to call it `targetRate`.

Reply:
@mwdchang makes sense - i've renamed to 'targetRate' and 'rate' respectively